src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/fa2e7bf5-9e59-4431-b016-a1d9d8ac6fce)

Reply:
not fixed